<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BmpReceive</class>
 <widget class="QDialog" name="BmpReceive">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recive</string>
  </property>
  <widget class="QLabel" name="lbl_image">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_error">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
